packages/@vuepress/theme-default/package.json

```diff
@@ -34,6 +34,7 @@
     "@vuepress/plugin-nprogress": "^1.0.3",
     "@vuepress/plugin-search": "^1.0.3",
     "docsearch.js": "^2.5.2",
+    "lodash": "^4.17.15",
```

Comment:
I think I remember Lodash being hard to tree-shake when the full package is loaded, even if you only pick import what you need. Do you know if this increases the size of the build?

If it does, I'd go for the micro-bundles:
- https://www.npmjs.com/package/lodash.isstring
- https://www.npmjs.com/package/lodash.isnil

Reply:
I'm using a VSCode plugin that tells me about the weight of my imports, everything seems to be ok 👍 :

![Capture d’écran 2019-08-11 à 19 31 32](https://user-images.githubusercontent.com/9840435/62837359-dd7ea600-bc6e-11e9-830f-219b631e8cf8.jpg)